src/components/register/RegisterForm.vue

```diff
@@ -0,0 +1,268 @@
+        <div class="field">
+            <label class="field__label" for="last_name">Last name</label>
+            <b-form-input
+                id="last_name"
+                type="text"
+                autocomplete
+                class="field__input"
+                @focus="resetError()"
+                @blur="check('last_name')"
+                v-model="lastname.value"
+            />
+        </div>
+        <div class="field">
+            <label class="field__label" for="birth">Date of birth</label>
+            <b-form-datepicker
+                id="birth"
+                type="text"
+                placeholder="DD-MM-YYYY"
+                autocomplete
+                class="mb-3"
+                @focus="resetError()"
+                @blur="check('birth')"
+                v-model="birthday.value"
+            />
+        </div>
+
+        <div class="field">
+            <label class="field__label" for="email">Email</label>
+            <b-form-input
+                id="email"
+                type="email"
+                autocomplete="username"
+                aria-describedby="email-constraints"
+                required
+                aria-required="true"
+                class="field__input"
+                @focus="resetError()"
+                @blur="check('email')"
+                v-model="email.value"
+            />
+            <p
+                id="email-constraints"
+                class="field__error"
+                v-if="hasError('email')"
+            >
+                Email is required
+            </p>
+        </div>
+        <div class="field">
+            <label class="field__label" for="password">Password</label>
+            <b-form-input
+                id="password"
+                type="password"
+                autocomplete="current-password"
+                aria-describedby="password-constraints"
+                required
+                aria-required="true"
+                class="field__input"
+                @focus="resetError()"
+                @blur="check('password')"
+                v-model="password.value"
+            />
+            <p
+                id="password-constraints"
+                class="field__error"
+                v-if="hasError('password')"
+            >
+                Password is required
+            </p>
+        </div>
+        <div class="field">
+            <label class="field__label" for="password2">Confirm password</label>
+            <b-form-input
+                id="password2"
+                type="password"
+                autocomplete="current-password"
+                aria-describedby="password-constraints"
+                required
+                aria-required="true"
+                class="field__input"
+                @focus="resetError()"
+                @blur="check('confirm_password')"
+                v-model="confirmpassword.value"
+            />
+            <p
+                id="password-constraints"
```

Comment:
```suggestion
                id="confirm-password-constraints"
```
On ne peut pas avoir 2 ids dans une même page